ExpenseTracker.Core/Models/Transactions/Exceptions/NotFoundTransactionException.cs

```diff
@@ -13,5 +13,9 @@ public class NotFoundTransactionException : Xeption
         public NotFoundTransactionException(Guid transactionId)
             : base(message: $"Couldn't find transaction with Id {transactionId}.")
```

Comment:
```suggestion
            : base(message: $"Transaction not found with Id {transactionId}.")
```

Reply:
or something other than Couldn't, just needs to be something more business appropriate.